api/v1/mongodbcommunity_types.go

```diff
@@ -485,6 +492,10 @@ func (m MongoDBCommunity) CurrentReplicas() int {
 	return m.Status.CurrentStatefulSetReplicas
 }
 
+func (m MongoDBCommunity) GetVersion() string {
```

Comment:
[nit] maybe this could be a but more specific (maybe `GetMongoDBVersion`) since we deal with a lot of different components that have different versions. (agent/mongodb/init containers and OpsManager with enterprise)

Reply:
Absolutely right!